api/testdata/complete-scenarios/test3.json

```diff
@@ -2638,7 +2638,7 @@
                                     "layout": "Address",
                                     "street": "Apple St",
                                     "city": "Berlin",
-                                    "zipcode": "12333",
+                                    "zipcode": "",
```

Comment:
I removed the zip code because international addresses do not have zip codes. The generated XML does not include the zip code `12333` that was previously in the test case.

My suspicion is that this data slipped in before we started to clear out irrelevant data when switching between branches. It was never caught as an error by e-QIP because it's not in the generated XML.